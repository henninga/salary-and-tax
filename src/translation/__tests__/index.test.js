import { toggleLocale } from '../index';


describe('translation', () => {
  describe('toggleLocale', () => {
    it('should set next locale', () => {
      expect(toggleLocale({locale: 'nb'})).toEqual('en');
      expect(toggleLocale({locale: 'en'})).toEqual('nb');
    });
    it('should default to nb', () => {
      expect(toggleLocale()).toEqual('nb');
      expect(toggleLocale({})).toEqual('nb');
      expect(toggleLocale({locale: 'unknown'})).toEqual('nb');
    });
  });
});
