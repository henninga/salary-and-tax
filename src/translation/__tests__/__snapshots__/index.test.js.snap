// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`translation getLocales should return all locales 1`] = `
Array [
  Object {
    "active": true,
    "label": "Norsk",
    "value": "nb",
  },
  Object {
    "active": false,
    "label": "English",
    "value": "en",
  },
]
`;

exports[`translation getMessages should return norwegian messages 1`] = `
Object {
  "salary.calculations.employer.fee": "Arbeidsgiveravgift",
  "salary.calculations.header": "Inntekstberegninger",
  "salary.calculations.income": "Inntekt",
  "salary.calculations.pension": "Pensjon",
  "salary.calculations.subcontractor.cut": "Andel til overleverandør",
  "salary.calculations.turnover": "Omsetning",
  "salary.calculations.vacation.savings": "Feriepenger",
  "salary.parameters.employer.fee": "Arbeidsgiveravgift",
  "salary.parameters.header": "Parametere",
  "salary.parameters.hour.rate": "Timepris",
  "salary.parameters.hours.per.year": "Timer per år",
  "salary.parameters.include.in.calculation": "Inkluder i utregningen",
  "salary.parameters.locale": "Språk",
  "salary.parameters.pension": "Pensjon",
  "salary.parameters.pension.step1": "(%) (1G-7.1G)",
  "salary.parameters.pension.step2": "(%) (7.1G-12G)",
  "salary.parameters.percentage": "(%)",
  "salary.parameters.subcontractor": "Overleverandør",
  "salary.parameters.subcontractor.cut": "Andel (%)",
  "salary.parameters.submit": "Beregn",
  "salary.parameters.turnover": "Omsetning",
  "salary.parameters.vacation.savings": "Feriepenger",
  "summary.after.tax": "Brutto lønn",
  "summary.common.income": "Alminnelig inntekt",
  "summary.header": "Sammendrag",
  "summary.income": "Inntekt",
  "summary.total.tax": "Skatt sammenlagt",
  "summary.turnover": "Omsetning",
  "tax.calculations.common.income": "Alminnelig inntekt",
  "tax.calculations.header": "Skatteberegninger",
  "tax.calculations.income": "Inntekt",
  "tax.calculations.income.tax": "Inntektsskatt",
  "tax.calculations.minimum.deduction": "Minstefradrag",
  "tax.calculations.social.security.deduction": "Trygdeavgift",
  "tax.calculations.step.1": "Trinnskatt trinn 1",
  "tax.calculations.step.2": "Trinnskatt trinn 2",
  "tax.calculations.step.3": "Trinnskatt trinn 3",
  "tax.calculations.step.4": "Trinnskatt trinn 4",
  "tax.calculations.total.tax": "Samlet skatt",
}
`;
